LayoutTests/fast/scrolling/mac/scrollbars/scrollbar-gutter-root-both-edges-expected.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
```

Comment:
We also need the tests with background image and abspos.

Reply:
This test utilized abspos (as do other wpts). I don't think there's a good way to do background image as I don't think there's a good way to expose the overhang like using scrollbar-gutter does.